src/Features/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptCompletionProvider.cs

```diff
@@ -19,7 +19,14 @@ public sealed override bool ShouldTriggerCompletion(SourceText text, int caretPo
         }
 
         internal sealed override bool ShouldTriggerCompletion(HostLanguageServices languageServices, SourceText text, int caretPosition, CompletionTrigger trigger, CompletionOptions options, OptionSet passThroughOptions)
-            => ShouldTriggerCompletionImpl(text, caretPosition, trigger, options.TriggerOnTypingLetters);
+        {
```

Comment:
We only need to trigger the completion of non-expanded items.

Reply:
Turns out that we didn't need to make any changes here and just verify that the correct `GetCompletionsMethod` is invoked. Functioning locally and doesn't require any changes in the TypeScript side.